physics.py

```diff
@@ -203,6 +231,14 @@ def __init__(self, physics_controller: PhysicsInterface, robot: MyRobot):
 
         self.imu = robot.chassis.imu.sim_state
 
+        self.injector = SparkMotorSim(
+            DCMotor.NEO550,
+            robot.injector_component.injector_1,
+            robot.injector_component.injector_2,
+            gearing=1 / 1,
+            moi=0.000105679992,  # TODO: measure
+        )
```

Comment:
This works but is technically wrong. The mass MOI is probably close enough (even though it was hallucinated by AI lol), but this doesn't take into account the gear reduction (which isn't accounted for in the encoder readings) because I can't do maths at 8pm